### DISCUSSION TITLE
Window and DataGrid

### DISCUSSION THREAD

**Discussion by hxhgxy** [CommunityToolkit/Windows#267] [Q&A]
I didn't find Window and DataGrid in the components. Would you please add them in next version?

Thanks

**Comment by Jay-o-Way** [comment]
Related to #269

**Comment by shadow-cs** [comment]
See https://github.com/CommunityToolkit/Windows/discussions/31#discussioncomment-5686828

**Comment by svsnux** [comment]
What is happening with the DataGrid? seems like the DataTable in the Labs was abandoned.

**Reply by Petrarca181** [reply]
> What is happening with the DataGrid? seems like the DataTable in the Labs was abandoned.

https://github.com/w-ahmad/WinUI.TableView
